### DISCUSSION TITLE
value of the square root of 2 ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#32] [Galaxy Qs] [answered]
What is the value of the square root of 2 ?

**Comment by EshayDev** [comment, marked as answer]
approximately 1.41421356

**Reply by Sriansh-raj** [reply]
Congratulations @EshayDev you just got a `Galaxy Brain` achievement. 🎉 
Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃

**Comment by sage-etcher** [comment]
the full number is irrational, but if we round to the 65th decimal place it would be ±1.41421356237309504880168872420969807856967187537694807317667973799

**Reply by Sriansh-raj** [reply]
Demn @sage-etcher you got it upto 65th decimal 😃

**Reply by Sriansh-raj** [reply]
But your answer needs to be marked as answer to get a `Galaxy Brain` 🙃

**Reply by Sriansh-raj** [reply]
@sage-etcher why don't you try answering other questions in [Galaxy Qs](https://github.com/Sriansh-raj/GitHub-Achievements/discussions/categories/galaxy-qs) 😃
